packages/smooth_app/lib/pages/onboarding/consent_analytics_page.dart

```diff
@@ -0,0 +1,175 @@
+// ignore_for_file: avoid_void_async
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:smooth_app/data_models/onboarding_loader.dart';
+import 'package:smooth_app/data_models/user_preferences.dart';
+import 'package:smooth_app/pages/onboarding/onboarding_flow_navigator.dart';
+
+import '../../database/local_database.dart';
```

Comment:
Could you use a package import here to be consistent?

Reply:
solved